tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/PackageCheckName.cs

```diff
@@ -0,0 +1,22 @@
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Defines the types of validation checks that can be run on SDK packages.
+/// </summary>
+public enum PackageCheckName
+{
+    /// <summary>
+    /// Run all available validation checks.
+    /// </summary>
+    All,
+    
+    /// <summary>
+    /// Run changelog validation check.
+    /// </summary>
+    Changelog,
+    
+    /// <summary>
+    /// Run dependency analysis check.
```

Comment:
Not for this PR, but I'd like to get a better idea of what this means. I don't think it translates 1:1 with other languages - perhaps it's very clear in Python what that means.

Reply:
It is different per lang we have a work item from the design meeting for each lang to figure out what it means for them -- we can update the description as we implement